docker/cassandra/cassandra.sh

```diff
@@ -32,15 +22,21 @@ JVM_OPTS="${JVM_OPTS} -Djava.library.path=${CASSANDRA_HOME}/lib/sigar-bin"
 JVM_OPTS="${JVM_OPTS} -XX:HeapDumpPath=/var/tmp/cassandra-`date +%s`-pid$$.hprof"
 
 
-# Read additional JVM options from jvm.options file
-JVM_OPTS="${JVM_OPTS} "$((sed -ne "/^-/p" | tr '\n' ' ') < ${CASSANDRA_CONF}/jvm.options)
-
-
-. ${CASSANDRA_CONF}/cassandra-env.sh
+# read additional JVM options from jvm.options files
+for options_file in "${CASSANDRA_CONF}/jvm.options" "${CASSANDRA_CONF}/jvm.options.d"/*.options
+do
+    JVM_OPTS="${JVM_OPTS} "$((sed -ne "/^-/p" | tr '\n' ' ') < "${options_file}")
+done
 
+# source additional environment settings
+for env_file in "${CASSANDRA_CONF}/cassandra-env.sh" "${CASSANDRA_CONF}/cassandra-env.sh.d"/*.sh
+do
+    . "${env_file}"
+done
 
 exec -a cassandra /usr/bin/java \
     -cp "${CASSANDRA_CLASSPATH}" \
+    -ea \   # beacuse turing this off is a bad, bad idea
```

Comment:
That comment may be defeating the escape of the newline

Reply:
@rs017991 yep. thanks for pointing that out.
I've fixed this in #92, but I'll back-port it.